tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -109,6 +112,46 @@ def run(self):
                 _cluster.current_job['status'] = "finished"
                 _cluster.is_managed = "yes"
                 _cluster.save()
+
+                # Creating a Job to do the profiling task. This job is
+                #  separate from the import flow.
+                if _cluster.volume_profiling_flag != "leave-as-is":
+                    if (_cluster.volume_profiling_state !=
+                            _cluster.volume_profiling_flag+"d"):
+                        _job_id = str(uuid.uuid4())
+                        params = {"TendrlContext.integration_id":
+                                      NS.tendrl_context.integration_id,
+                                  "Cluster.volume_profiling_flag":
+                                      _cluster.volume_profiling_flag
+                                  }
+                        payload = {"tags": ["provisioner/%s" %
+                                            NS.tendrl_context.integration_id],
+                                   "run": "gluster.flows.EnableDisableVolume"
+                                          "Profiling",
+                                   "status": "new",
+                                   "parameters": params,
+                                   "type": "sds",
+                                   "created_at": (datetime.datetime.
+                                                  utcnow().strftime
+                                                  ('%Y-%m-%d %H:%M:%SZ')),
+                                   "created_from": "BackendSpecialCase"
+                                   }
+                        NS.tendrl.objects.Job(
+                            job_id=_job_id,
+                            status="new",
+                            payload=payload
+                        ).save()
+                        logger.log(
+                            "info",
+                            NS.publisher_id,
+                            {"message": "Created job ( %s ) for profiling. "
```

Comment:
`Created job (%s) for enable/disable profiling. The job could be tracked independently once this import job is completed.`
Does this look better?

Reply:
yes, Will do the change